### ISSUE TITLE
Issue: Sending Image through FormData

### ISSUE THREAD

**Issue by emilianofunk** [homer0/wootils#45]
Hello there!
I have a use case where I need to upload an image using the FormData class and going through the fetch from the ApiClient, but I found that there was no way to perform the request since it seems the ApiClient is "Stringify-ing" the formData object, then sending an empty object/json.

https://github.com/homer0/wootils/blob/master/shared/apiClient.js#L313-L319

We did something like this to overcome this issue:

```js
if (opts.body) {
  // Let's first check if there are headers and if a `Content-Type` has been set.
  let hasContentType = false;
  if (opts.headers) {
    hasContentType = Object.keys(opts.headers)
    .find((name) => name.toLowerCase() === 'content-type');
  } else {
    opts.headers = {};
  }
  // If the body is an object...
  if (typeof opts.body === 'object') {
    const contentTypeHeader = opts.headers[hasContentType] || '';

    // And if no `Content-Type` was defined, let's assume is a JSON request.
    if (!hasContentType) {
      opts.headers['Content-Type'] = 'application/json';
    }

    if (!hasContentType || contentTypeHeader.includes('json')) {
      opts.body = JSON.stringify(opts.body);
    }
  }
  ...
```

Regards 😛

**Comment by homer0** [comment]
On it.

**Comment by homer0** [comment]
@emilianofunk I merged #47 and released [v2.6.2](https://github.com/homer0/wootils/releases/tag/2.6.2). Let me know if it works for your case.

**Comment by emilianofunk** [comment]
It works like a charm!
Thanks dude!
